python-sdk/example_dags/example_amazon_s3_snowflake_transform.py

```diff
@@ -58,6 +58,7 @@ def example_amazon_s3_snowflake_transform():
             schema=os.environ["SNOWFLAKE_SCHEMA"],
         ),
         conn_id="snowflake_conn",
+        temp=True,
```

Comment:
if you want to create tmp table then the below would be sufficient or do we need the table name for some reason? in our context the tmp usually contains prefix _tmp 
```
input_table_1 = Table(
        metadata=Metadata(
            database=os.environ["SNOWFLAKE_DATABASE"],
            schema=os.environ["SNOWFLAKE_SCHEMA"],
        ),
        conn_id="snowflake_conn",
)
```

Reply:
we would not need the table name. Even _tmp should suffice. Let me change that and run once the dag.

Reply:
I think we need named temp tables in this case. Also this `temp=True` is valid way to create temp tables as suggested in our docs too: https://astro-sdk-python.readthedocs.io/en/stable/guides/concepts.html